tensorflow/compiler/mlir/tosa/transforms/legalize_utils.cc

```diff
@@ -285,7 +285,17 @@ Value getTosaConst8bitTable(PatternRewriter& rewriter, Operation* op,
   for (int32_t i = -128; i < 128; i++) {
     double dequantized = input_scale * (i - input_zp);
     double transformed = func(dequantized);
+
+    std::feclearexcept(FE_ALL_EXCEPT);
```

Comment:
Could this be avoided? E.g., could we detect rather than recover from?

Reply:
I think I've got a better option now. In theory the existing code could do something if NaN or infinities were present, but I think we're better off avoiding those when generating table values.

We can pre-test overflow with this code:
```
double max = (output_scale > 1.0) ? DBL_MAX : DBL_MAX * output_scale;
if (transformed > max) {
  rescaled = INT8_MAX;
}
else {
  rescaled = std::llround(transformed / output_scale);
}
```

Reply:
New patch set pushed.